pkg/server/apikeys/service_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package apikeys_test
+
+import (
+	"testing"
+
+	t_apikeys "github.com/daytonaio/daytona/internal/testing/server/apikeys"
+	"github.com/daytonaio/daytona/pkg/apikey"
+	"github.com/daytonaio/daytona/pkg/server/apikeys"
+	"github.com/stretchr/testify/suite"
+)
+
+var clientKeyNames []string = []string{"client1", "client2", "client3"}
+var projectKeyNames []string = []string{"project1", "project2"}
+
+var expectedKeys []*apikey.ApiKey
```

Comment:
No need for the `expectedKeys` to be "global". They can be defined in each of the tests. It also eliminates the need for the `SetupTest` function to reset them.